app/util/metrics/MultichainAPI/networkMetricUtils.ts

```diff
@@ -0,0 +1,71 @@
+import {
+  CaipChainId,
+  isCaipChainId,
+  toCaipChainId,
+  isHexString,
+} from '@metamask/utils';
+import { selectNetworkConfigurations } from '../../../selectors/networkController';
+import { store } from '../../../store';
+import { UserProfileProperty } from '../UserSettingsAnalyticsMetaData/UserProfileAnalyticsMetaData.types';
+import { convertHexToDecimal } from '@metamask/controller-utils';
+
+/**
+ * Converts a chain ID to CAIP format.
+ * @param chainId - The chain ID to convert, either as a CAIP chain ID or string.
+ * @returns The chain ID in CAIP format.
+ */
+function caipifyChainId(chainId: CaipChainId | string): CaipChainId {
+  const _chainId = isHexString(chainId)
+    ? convertHexToDecimal(chainId)
+    : chainId;
+
+  return isCaipChainId(_chainId)
+    ? _chainId
+    : toCaipChainId('eip155', _chainId.toString());
+}
+
+/**
+ * Gets an array of all configured network chain IDs converted to CAIP format.
+ * @returns An array of CAIP chain IDs for all configured networks.
+ */
+export function getConfiguredChainIdsCaipified(): CaipChainId[] {
```

Comment:
is this **all** networks or just EVM networks?
```suggestion
export function getConfiguredCaipChainIds(): CaipChainId[] {
```